MonoGame.Framework/Input/GamePad.Android.cs

```diff
@@ -258,8 +258,17 @@ internal static bool OnGenericMotionEvent(MotionEvent e)
 
             gamePad._leftStick = new Vector2(e.GetAxisValue(Axis.X), -e.GetAxisValue(Axis.Y));
             gamePad._rightStick = new Vector2(e.GetAxisValue(Axis.Z), -e.GetAxisValue(Axis.Rz));
-            gamePad._leftTrigger = e.GetAxisValue(Axis.Ltrigger);
-            gamePad._rightTrigger = e.GetAxisValue(Axis.Rtrigger);
+
+            if (e.Device.Name == "Xbox Wireless Controller")
```

Comment:
this test could be done once (somewhere...) and the Axis to use would be stored in a class variable to be used here.

Reply:
Absolutely correct, could be optimized via a configuration that allows more known gamepads to be recognized. Still this works better than the current implementation, that doesn't work with one of the most wide-spread controllers to date.